content/react/en/creating-addons.md

```diff
@@ -0,0 +1,429 @@
+---
+title: "Creating addons"
+tocTitle: "Creating addons"
+description: "Learn how to build your own addons that will super charge your development"
+commit: "dac373a"
+---
+
+# Creating addons
+
+Storybook's addon architecture allows you to expand it's feature set and customize it to your needs.
+
+Writing your own addon may sound a bit daunting, but as you'll see in this guide, it can be quite simple.
+
+We're going to write an addon together, first let's scope what our addon will do:
+
+## The addon we're going to write
```

Comment:
Lets include a screenshot of the finished product to whet people's appetites. I propose a screenshot that includes the taskbox example that we build in LSB with the design-assets addon showing a design that backs one of the stories.

Reply:
I think a bit stylised promo picture would be best, I have no good example of a design asset + story.

To test this addon we have it in the monorepo, but there are no design assets that would make sense to display.

<img width="1920" alt="Screenshot 2019-09-23 at 16 08 13" src="https://user-images.githubusercontent.com/3070389/65433062-9b738300-de1c-11e9-8aa4-859fa16f2dc5.png">

@domyen do you think a picture like the ones here:
https://storybook.js.org/addons/

could be created?

Reply:
@ndelangen The above ss is too disconnected, we want this to make sense to LSB users. Do this:
1. Add the design-assets addon to taskbox
2. Navigate to a good looking story in taskbox
3. Take a ss of the component
4. Link to it in design-assets
5. Take a screenshot (as above) of the design asset sitting next to the story. They should look the same.

Reply:
What's 'taskbox'?

Reply:
Taskbox is the name of the app that Learn Storybook teaches people how to build

Reply:
👍